src/js/helpers/glossaryHelper.js

```diff
@@ -15,3 +15,17 @@ export const fetchSearchResults = (params) => apiRequest({
     method: 'post',
     data: params
 });
+
+export const getNewUrlForGlossary = (pathname, url, queryParams = '') => {
```

Comment:
Do we have comments explaining this function?

Do we have tests for this?

Reply:
@jonhill13 writing them right now!

Reply:
@jonhill13 pushed tests.